### DISCUSSION TITLE
Welcome to jekyll-klise Discussions!

### DISCUSSION THREAD

**Discussion by piharpi** [piharpi/jekyll-klise#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zeyu-Li** [comment]
👋 hey, don't want to start an issue over this but if someone could do a tutorial on how to link this up with google analytics here or on the wiki tab, it would be much appreciated! 
Thanks in advance
